src/main/java/by/exadel/internship/entity/user/User.java

```diff
@@ -1,20 +1,20 @@
 package by.exadel.internship.entity.user;
-import by.exadel.internship.dto.enums.Technology;
+
 import by.exadel.internship.dto.enums.UserRole;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+
 import javax.persistence.*;
-import java.util.List;
 import java.util.UUID;
 
 @Entity
-@Table(name = "user")
+@Table(name = "user", schema = "public")
```

Comment:
what purposes do you specify schema for?

Reply:
IT WAS Grammar exception from hibrnate, and it was pretty strange, because everything was right, and schema was decision from net about this problem.

Reply:
I think that because postgres has another user table and schema = "public" resolves that conflict

Reply:
yes, but we took desicion, how i remember, about singular naming. @kzaretskaya  maybe we should change this?

Reply:
What will be changed? to plural naming? no. This entity is already in branch development, how does it work there? If problem appeared on your branch it need to figure out cause.

Reply:
![image](https://user-images.githubusercontent.com/65280608/114516336-f9a5c800-9c45-11eb-9b74-bf8fb9d57457.png)
this is a working option also

Reply:
https://stackoverflow.com/questions/22256124/cannot-create-a-database-table-named-user-in-postgresql

user is a reserved word and it's usually not a good idea use reserved words for identifiers (tables, columns).

Reply:
I think it's not a good idea to wrap with quotes. This problem had to be appeared when the entity was created (probably it's not tested).
My suggestion to rename table to user_detail.
@viktarkolbik @SergeyKoval what do you think?

Reply:
Agree with @kzaretskaya

Reply:
done 
![image](https://user-images.githubusercontent.com/65280608/114573224-dba98900-9c80-11eb-9004-4ce851fc1c11.png)